### ISSUE TITLE
validate that timestamps on servers are within reasonable amounts (seconds)

### ISSUE THREAD

**Issue by thoraxe** [openshift/openshift-ansible#961]
A user reported an issue where the installer happily marched along even with the servers having timestamps that were more than 10 minutes out.

Should we validate that server TZs and times are reasonably in sync?

**Comment by detiber** [comment]
@thoraxe it's a tricky issue to solve that depending on the cluster size we may not have a good way to handle it. Specifically, when the number of host instances exceeds the number of ansible forks.

We could verify each server time against the client time, but that has little relevance to the service working.

We could also require that the user set an ntp source and force a sync of the local time off of that, but that seems like we may be reaching beyond our scope...

**Comment by thoraxe** [comment]
I'm not saying we have to solve it. I'm just saying that we should check/verify/warn and then bomb if not.

Since we are generating and consuming SSL certificates, there can be issues with time sensitivity. For example, if we generate a cert whose validity starts at 10:00:00AM but one of the machines has a clock set to 9:50:00AM then the cert it is going to talk to isn't even valid for another ten minutes. This can cause issues.

So, basically, before doing anything, iterate over all hosts, check that times are within a few seconds, and abort if not. Or something.
